### ISSUE TITLE
[Broken Website] [Minor] Google Docs, Sheets, Slides

### ISSUE THREAD

**Issue by Kiansjet** [darkreader/darkreader#5893]
# Broken Website

## Website
https://docs.google.com/- (Google document editor)
You'll have to create a GDoc (Docs, Sheets, Slides) to observe this.

## How to Reproduce
1. Click the "present to a meeting" button in the top right button cluster.
2. The "present using a meeting code" keyboard icon doesn't render correctly with the material icons font.
3. Click "present using a meeting code".
4. The back arrow icon also doesn't render correctly with the material icons font.

## Expected behavior
Material icons should be displayed in place of those icon names.

## Actual behavior
Material icon names are displayed instead of the actual icons.

## Screenshots
![image](https://user-images.githubusercontent.com/13376313/119851618-63f78c80-bec3-11eb-8c6a-81f9671f674e.png)
![image](https://user-images.githubusercontent.com/13376313/119851662-71ad1200-bec3-11eb-9b4e-29df325bcc12.png)
![image](https://user-images.githubusercontent.com/13376313/119851811-95705800-bec3-11eb-8837-ed5443b2d4da.png)
![image](https://user-images.githubusercontent.com/13376313/119851870-a5883780-bec3-11eb-9882-d1cc0ef18138.png)

## System Information

- OS: Windows 10 Home build 19043.1023
- Browser: Chrome 90.0.4430.212
- Dark Reader Version: 4.9.32

## Additional context
- Issue is not encountered or resolved when toggling DR on the loaded page (turning DR off without reloading doesn't solve the issue and turning DR on without reloading doesn't cause it)

**Comment by Gusted** [comment]
> I've also encountered this problem. All of the icons (except for the first column) of [Google's Material icon font](https://material-icons.github.io/material-icons-font/samples/test.html) will not load if the extension is enabled for that website.

![image](https://user-images.githubusercontent.com/25481501/137959856-6d09774b-5f7d-44d4-b035-a9e86ad1b559.png)
Seems all good on my side. the icons are pure Unicode characters, not really something Dark Reader messes with.

**Comment by Gusted** [comment]
Yeah the font's option can cause some issues with specific fonts like the icons font from Google.

**Comment by fabform** [comment]
Handy to [find google sheet id](https://fabform.io/a/how-to-find-google-sheet-id)

**Comment by Kiansjet** [comment]
> You are right, disabling the 'system-ui' font seems to fix the issue, which apparently was enabled

Hello!

@stronk-dev can you clarify as to how you went about disabling this 'system-ui' font? I can't find a relevant control in the extension settings, nor a mention of such a string in the devtools.

**Comment by stronk-dev** [comment]
There's a font setting within the drop-down menu. If you're not running Firefox, `system-ui` might not even pop up as a available font.
